proto/v1/common.proto

```diff
@@ -103,3 +105,17 @@ message Signature {
   //   * "nih:///sha-256;703f68f42aba2c6de30f488a5ea122fef76324679c9bf89791ba95a1271589a5"
   string public_key_id = 2;
 }
+
+// MUST match https://github.com/secure-systems-lab/dsse/blob/master/envelope.proto.
+// An authenticated message of arbitrary type.
+message Envelope {
+  message Signature {
```

Comment:
Note that common.proto already has a Signature field. So I moved the Envelope Signature inside the Envelope message, which is technically different than https://github.com/secure-systems-lab/dsse/blob/master/envelope.proto.

Reply:
Yes, this sounds great. In v2, we'll make sure to reconcile these.